lxd/storage/drivers/generic_vfs.go

```diff
@@ -496,37 +492,36 @@ func genericVFSBackupVolume(d Driver, vol Volume, tarWriter *instancewriter.Inst
 					exclude = append(exclude, blockPath)
 				}
 
-				logMsg := "Copying virtual machine config volume"
-				if vol.volType == VolumeTypeCustom {
-					logMsg = "Copying custom config volume"
-				}
+				if v.IsVMBlock() {
+					logMsg := "Copying virtual machine config volume"
 
-				d.Logger().Debug(logMsg, log.Ctx{"sourcePath": mountPath, "prefix": prefix})
-				err = filepath.Walk(mountPath, func(srcPath string, fi os.FileInfo, err error) error {
-					if err != nil {
-						return err
-					}
+					d.Logger().Debug(logMsg, log.Ctx{"sourcePath": mountPath, "prefix": prefix})
+					err = filepath.Walk(mountPath, func(srcPath string, fi os.FileInfo, err error) error {
+						if err != nil {
+							return err
+						}
 
-					// Skip any exluded files.
-					if shared.StringHasPrefix(srcPath, exclude...) {
-						return nil
-					}
+						// Skip any exluded files.
+						if shared.StringHasPrefix(srcPath, exclude...) {
+							return nil
+						}
 
-					name := filepath.Join(prefix, strings.TrimPrefix(srcPath, mountPath))
-					err = tarWriter.WriteFile(name, srcPath, fi, false)
+						name := filepath.Join(prefix, strings.TrimPrefix(srcPath, mountPath))
+						err = tarWriter.WriteFile(name, srcPath, fi, false)
+						if err != nil {
+							return errors.Wrapf(err, "Error adding %q as %q to tarball", srcPath, name)
+						}
+
+						return nil
+					})
 					if err != nil {
-						return errors.Wrapf(err, "Error adding %q as %q to tarball", srcPath, name)
+						return err
 					}
-
-					return nil
-				})
-				if err != nil {
-					return err
 				}
 
 				name := fmt.Sprintf("%s.%s", prefix, genericVolumeBlockExtension)
 
-				logMsg = "Copying virtual machine block volume"
+				logMsg := "Copying virtual machine block volume"
```

Comment:
this message needs to change based on whether the volume is VMBlock or not, similar to below:

```go
			logMsg := "Unpacking virtual machine block volume"
			if vol.volType == VolumeTypeCustom {
				logMsg = "Unpacking custom block volume"
			}
```

Reply:
It already does.

Reply:
So it does, dont know why I didn't see that.